File: Cards/Deuce.lua
local ____lualib = require("lualib_bundle")
local __TS__Class = ____lualib.__TS__Class
local __TS__ClassExtends = ____lualib.__TS__ClassExtends
local ____exports = {}
local ____Enums = require("Enums")
local Ranks = ____Enums.Ranks
local ____Card = require("Cards.Card")
local Card = ____Card.default
____exports.default = __TS__Class()
local Deuce = ____exports.default
Deuce.name = "Deuce"
__TS__ClassExtends(Deuce, Card)
function Deuce.prototype.____constructor(self, gameManager, suit)
    Card.prototype.____constructor(
        self,
        gameManager,
        suit,
        Ranks.DEUCE,
        1,
        0,
        "Deuce"
    )
end
return ____exports

File: Cards/Deuce.ts
import { Ranks, Suits } from "../Enums";
import GameManager from "../GameManager";
import Card from "./Card";

export default class Deuce extends Card {
  constructor(gameManager: GameManager, suit: Suits) {
    super(
      gameManager,
      suit,
      Ranks.DEUCE,
      1,
      0,
      "Deuce",
      "Assets/Images/DeuceRank.png",
      {
        edelName: "The Chosen",
        edelPower: 16,
        edelValue: 11,
        edelRankAssetPath: "Assets/Images/ChosenRank.png",
      }
    );
  }
}
